modules/artifact/src/main/java/org/mule/module/artifact/classloader/IBMMQResourceReleaser.java

```diff
@@ -209,89 +195,61 @@ private Set processJULKnownLevels(
   }
 
   /**
-   * Kills the ThreadPool master thread for IBM classes.
-   * @param thread
-   */
-  private void killThreadPoolMaster(Thread thread) {
-    Class<? extends Thread> threadClass = thread.getClass();
-    boolean shouldCleanThread = isThreadFromCurrentClassLoader(threadClass);
-    if (shouldCleanThread) {
-      try {
-        Method closeMethod = threadClass.getDeclaredMethod("close");
-        closeMethod.setAccessible(true);
-        closeMethod.invoke(thread);
-        thread.interrupt();
-      } catch (Throwable e) {
-        LOGGER.warn("Caught exception when closing the '" + JMSCC_THREAD_POOL_MASTER_NAME + "' Thread: {}", e.getMessage(), e);
-      }
-    }
-  }
-
-
-  /**
-   * Clears the JmsTls static final references.
+   * The JmsTls class keep several references in a private static final field
+   * This references avoid the proper ClassLoader disposal.
+   * This method performs the JmsTls Class cleanup.
    */
   private void cleanJmsTls() {
-    /*
-    The JmsTls class keep several references in a private static final field
-    This references avoid the proper ClassLoader disposal.
-     */
     try {
-      //JmsTls
-      Class myClass = Class.forName("com.ibm.msg.client.jms.internal.JmsTls", false, driverClassLoader);
-      clearPrivateStaticField(myClass, "myInstance", true);
+      Class jmsTlsClass = loadClass(IBM_MQ_JMS_TLS_CLASS, driverClassLoader);
+      setStaticFieldValue(jmsTlsClass, "myInstance", null, true);
     } catch (Exception ex) {
       LOGGER.warn("Caught Exception when clearing JmsTls: {}", ex.getMessage(), ex);
```

Comment:
this message shoudn't be about catching a generic exception. If the class is not there, it's probably because the user is using a driver version which no longer uses it. The releaser should log that instead. Also, don't do a big try with this catch at the end. If this exception is to be raised by the `Class.forname` method, then have a try with that statement only and catch it right there. This comment applies all over the place